Documentation/compatibility/aspnet-472-compat-doc.md

```diff
@@ -0,0 +1,32 @@
+## Appsetting "dataAnnotations:dataTypeAttribute:disableRegEx" is On by Default in .NET 4.7.2
+
+### Scope
+Minor 
+
+### Version Introduced
+.NET 4.6.1
+
+### Version Reverted
+.NET 4.7.2
+
+### Change Description
+In 4.6.1, an appsetting (**"dataAnnotations:dataTypeAttribute:disableRegEx"**) was introduced which allows users to disable the use of Regular Expressions (RegExs) in data type attributes (such as EmailAddressAttribute, UrlAttribute, and PhoneAttribute) in System.ComponentModel.DataAnnotations.  This helps to reduce security vulnerability such as avoiding the possibility of Denial of Service attack using specific Regular Expressions.
```

Comment:
fence the setting (that is, use \` to begin and end its name) rather than boldface.